### ISSUE TITLE
Addon Error: Could Not Be Constructed

### ISSUE THREAD

**Issue by quartztester** [RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK#24]
After downloading this plugin the attached code appears. This issue seemed to pop up within the past day or two. 

```
AutoIdleOnAFK: Could not be constructed.
    at Object.initializeAddon (<anonymous>:4:306851)
    at Object.loadAddon (<anonymous>:4:268341)
    at Object.loadAddon (<anonymous>:4:305889)
    at FSWatcher.<anonymous> (<anonymous>:4:266190)
```

**Comment by RoguedBear** [comment]
I fixed the issue yesterday in [v0.3.1](https://github.com/RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK/releases/tag/v0.3.1), but havent seen the update notification in my discord too till now. 

I guess the update needs to be approved by the better discord admin team, and maybe then BD will show the update notification.

duplicate of #22 , but i'll close it when i myself get the update notification too. (i'm running the previous version to check if the update went through)

**Comment by RoguedBear** [comment]
To resolve construction error on your end, you need to redownload the fixed version of the plugin [from here](https://github.com/RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK/releases/tag/v0.3.1).
BetterDiscord does not show update banners for construction errors
